AppController/djinn.rb

```diff
@@ -1621,6 +1626,12 @@ def update(apps, secret)
        end
     end
 
+    RESERVED_APPS.each { |app|
```

Comment:
`reserved_app` would be clearer than `app` so that it's not assumed to be a member of `apps`.